run_configs/gcom/gcom_build_steps.py

```diff
@@ -15,7 +15,7 @@
 
 
 def common_build_steps(fpic=False) -> List[Step]:
-    steps: List[Step] = [
+    steps = [
```

Comment:
Not sure how removing the type hinting makes it easier to run.

Reply:
It was intended to make it easier to read for someone who doesn't know Python at all. We wanted it super-lightweight.